pages/accommodation.md

```diff
@@ -0,0 +1,89 @@
+title: Accommodation
+---
+
+# Hotels
+
+## Recommended partner hotels
+
+The hotels listed in this section have given us preferential rates or terms or both. Their allocation of rooms for
+PyCon UK attendees is likely to be exhausted well before the conference, so **you are strongly advised to book as early
+as you can**.
+
+Note that depending upon availability you may well be able to find cheaper rates than these, but on
+less flexible terms (changes and cancellation not permitted).
+
+Other hotels are listed after these.
+
+Cardiff is a small city, and the weekend of PyCon UK will be busy. The longer you leave it to book your accommodation, the more difficult it will be to get it at a good price.
+
+**Important**: if you find that the prices your are offered don't match the rates advertised below, that means that
+they are simply no longer available. We don't have a contract with the hotels, and once the room allocations are gone,
+they're gone.
+
+#### Sleeperz Hotel
+
+[Sleeperz Hotel](http://www.sleeperz.com) (tel +44 2920 478747); [email Sleeperz Hotel
+reservations](mailto:reservations@sleeperzcardiff.com)) is a comfortable budget hotel, a minute's walk from the station and 15
+minutes' walk from City Hall.
+
+Please quote the *PYCON SELECT* rate when booking (phone or email). The negotiated PyCon UK rates include breakfast, in a
+double or twin room.
+
+## Other hotels
+
+These hotels haven't given us special rates, but are conveniently located and inexpensive.
+
+#### Royal Hotel
+
+The [Royal Hotel](http://www.royalhotelcardiff.com) (tel +44 2920 550755; [email Royal Hotel
+reservations](mailto:reservations@theroyalhotel.uk.com)) is perfectly-placed, mid-way between City Hall (about 10
+minutes' walk) and Cardiff Central Railway station.
+
+#### Angel Hotel
+
+The [Angel Hotel](https://www.thehotelcollection.co.uk/hotels/cardiff-angel-hotel) (tel +44 2921 672322); [email Angel
+Hotel reservations](mailto:angel@thehotelcollection.co.uk)) is close to City Hall (about 10 minutes' walk) and Cardiff
+Central Railway station (about 15 minutes' walk).
+
+#### Sandringham Hotel
+
+The [Sandringham Hotel](http://sandringham-hotel.com) (tel +44 2920 232161); [email Sandringham Hotel
+reservations](mailto:mm@sandringham-hotel.com)) is mid-way between City Hall (about 10
+minutes' walk) and Cardiff Central Railway station.
+
+Please note that the hotel has five floors **but does not have a lift**. It's **not** suitable for people with reduced
+mobility.
+
+Please use the code *trip1* when booking (phone or email).
```

Comment:
Mmmm. I stayed there last year, and this was a generic voucher (although it is a hangover from last year). I’ll remove it for now.